### DISCUSSION TITLE
Manage Tasmota over the internet

### DISCUSSION THREAD

**Discussion by DiegoB80** [arendst/Tasmota#22101] [Ideas]
"It would be great to have the ability to manage a Tasmota device (via http) remotely (over the internet) without the need to create NAT on the router, static public IP, DDNS....
The idea would be to release a cloud service that acts as a reverse proxy (see for example https://www.socketxp.com/ or https://www.remote.it)."

**Comment by tsmoneta** [comment]
I just use a VPN (Tailscale or Wireguard) to remotely access everything on the LAN.

**Reply by sfromis** [reply]
As Tasmota can also support a MQTT-TLS setup, having it connect to a trusted broker (meaning one with more security than merely the encrypted transport), could also be an already supported option. But yeah, accessing the local network via a VPN is a good general approach.

(I'd never want to have a Tasmota device exposed to incoming connections from the Internet, meaning not even considering to have the lack of security with NAT, public IP, etc)
